src/__tests__/Voice.test.ts

```diff
@@ -30,10 +30,34 @@ describe("Voice operations", () => {
     Aflr.configure({ apiKey, debug });
   });
 
-  test("It should list all of the scripts and find the created one", async () => {
+  let allVoicesCount: number;
+
+  test("It should list all of the voices", async () => {
     try {
       const voices: any = await Voice.list();
       expect(Array.isArray(voices.voices)).toBe(true);
+      allVoicesCount = voices.voices.length;
+    } catch (e) {
+      console.error(e);
+      throw new Error("test failed");
+    }
+  });
+
+  test("It should list all of the voices that match the filtering parameters", async () => {
+    try {
+      const voices: any = await Voice.list({ tags: "fun", gender: "male", language: "english" });
+      expect(Array.isArray(voices.voices)).toBe(true);
+      expect(voices.voices.length).toBeLessThanOrEqual(allVoicesCount);
+    } catch (e) {
+      console.error(e);
+      throw new Error("test failed");
+    }
+  });
+
+  test("It should list all of the available filtering parameters", async () => {
+    try {
+      const parameters: any = await Voice.parameters();
+      expect(parameters !== null && typeof parameters === "object").toBe(true);
```

Comment:
you can remove the !== null I think if you have the object type.

and you can do better than this here in the test, you know the return type. you can say it must include everything in the IVoiceFilteringBody